pyocd/rtos/freertos.py

```diff
@@ -505,4 +509,25 @@ def get_is_running(self):
             return False
         return self._target_context.read32(self._symbols['xSchedulerRunning']) != 0
 
-
+    def _get_elf_symbol_size(self, name, addr, defValue):
```

Comment:
Please rename `defValue` to `calculated_size` to follow PEP8 naming of parameters and make the parameter's purpose more apparent. (At some point I'd like to turn on PEP8 checks in CI, so this would fail.)

Reply:
Okay